src/test/java/org/folio/edge/rtac/RtacHandlerTest.java

```diff
@@ -0,0 +1,192 @@
+package org.folio.edge.rtac;
+
+import static org.apache.http.HttpStatus.SC_NOT_ACCEPTABLE;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.folio.edge.core.Constants.APPLICATION_JSON;
+import static org.folio.edge.core.Constants.APPLICATION_XML;
+import static org.folio.edge.core.Constants.TEXT_PLAIN;
+import static org.folio.edge.rtac.utils.RtacUtils.composeMimeTypes;
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import java.io.IOException;
+import org.apache.http.HttpHeaders;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.edge.rtac.model.Instances;
+import org.folio.edge.rtac.utils.RtacMimeTypeEnum;
+import org.junit.Test;
+
+public class RtacHandlerTest extends MainVerticleTest {
+
+  private static final Logger logger = LogManager.getLogger(RtacHandlerTest.class);
+
+  /**
+   * Tests when there is no "Accept" header then response returns in XML by default.
+   *
+   * @throws IOException
+   */
```

Comment:
What information does this comment block provide that the name of the method and it's signature does not?

Reply:
Hm, I though that possibly human-readable way of providing comments will help quickly understand purpose of test case. I'll fix it.

Reply:
It might well do that. I would only include it in circumstances where it provides more information than the test name itself (which I would hope was human readable)

Reply:
You right, sometime extra information is unneeded, especially when method has self explanation name.